GitCommands/RevisionReader.cs

```diff
@@ -175,8 +175,11 @@ public sealed class RevisionReader : IDisposable
             return new GitArgumentBuilder("log")
             {
                 "-z",
-                branchFilter,
                 $"--pretty=format:\"{FullFormat}\"",
+                {
```

Comment:
Any reason to switch order?

I would pprefer to move this option close to the other where branchFilter is handled (either here or to line 194)

Reply:
> Any reason to switch order?

No reason, my mistake. Moved back to where it was.